src/http_api_handler.erl

```diff
@@ -216,3 +267,60 @@ maybe_add_sum(Name, DataPoints, histogram, Payload) ->
     [Payload | [Name, <<"_sum ">>, ioize(Mean * N), <<"\n">>]];
 maybe_add_sum(_Name, _DataPoints, _Type, Payload) ->
     Payload.
+
+get_sessions() ->
+    F = ets:fun2ms(fun ({{_SockName, {Id, _, _}}, Pid})
+                         when (Id == imsi) or (Id == imei) ->
+                           Pid
+                   end),
+    ets:select(gtp_context_reg, F).
+
+find_session(Id, BearerId) ->
+    BId = case BearerId of
+              undefined -> 5;
+              _ -> BearerId
+          end,
+    Sockets = application:get_env(ergw, sockets, []),
+    find_session(Sockets, Id, BId).
+
+find_session([], _Id, _BearerId) -> [];
```

Comment:
lookup/get methods in ergw return ```undefined``` or ```{error, not_found}``` in this case. Please use either one of these as well.